src/main/cli/settings.c

```diff
@@ -1223,7 +1223,7 @@ const clivalue_t valueTable[] = {
     { PARAM_NAME_TPA_MODE,          VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_TPA_MODE }, PG_PID_PROFILE, offsetof(pidProfile_t, tpa_mode) },
 #endif
     { PARAM_NAME_TPA_RATE,          VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, TPA_MAX}, PG_PID_PROFILE, offsetof(pidProfile_t, tpa_rate) },
-    { PARAM_NAME_TPA_BREAKPOINT,    VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_PID_PROFILE, offsetof(pidProfile_t, tpa_breakpoint) },
+    { PARAM_NAME_TPA_BREAKPOINT,    VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { 1000, 1990 }, PG_PID_PROFILE, offsetof(pidProfile_t, tpa_breakpoint) },
```

Comment:
This is very prone to induce a rounding bug in `pid_init.c`. Try this instead in combination with a review comment in `pid_init.c`.
```suggestion
    { PARAM_NAME_TPA_BREAKPOINT,    VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { PWM_RANGE_MIN, PWM_RANGE_MAX }, PG_PID_PROFILE, offsetof(pidProfile_t, tpa_breakpoint) },
```

Reply:
@KarateBrot : IMO problem is that throttle is in already in `0-1` range when TPA is applied. Using `PWM_RANGE_x` will probably break mapping to user-set ranges.

Reply:
Would it be better to just map `[PWM_PULSE_MIN, PWM_PULSE_MAX]` to `[0, 1]`?

Reply:
@KarateBrot please let me know your thoughts on this point?
I'm happy to use `PWM_RANGE_MIN` as the lower bound.
If we use `PWM_RANGE_MAX` as the upper bound, pidRuntime.tpaBreakpoint can be 1.0f, which results in a divide by zero when we calculate `pidRuntime.tpaMultiplier`.
Very high values for the breakpoint are inappropriate, because they cause a sudden-onset TPA effect only at the very extreme of throttle position. Even 1990 is 'too high to be usable'.  1900 is too high also.  Our default is way down at 1350 and I personally use 1150 on many quads to get useful mid-range attenuation.

Reply:
Sorry, my fault. I mixed PWM_PULSE_MIN and PWM_RANGE_MIN.
PWM_RANGE_MIN,PWM_RANGE_MAX is fine.

IMO TPA breakpoint won't match throttle value exactly (if other throttle modifications are applied before TPA), but it has negligible consequences.